js/script.js

```diff
@@ -1,16 +1,18 @@
 // Book Class: Represents a Book
 class Book {
-  constructor(title, author, isbn) {
+  constructor(title, author, email, phone, isbn) {
```

Comment:
Consider making email and phone number private as these are not meant to be accessed  or modified outside the class.
For that getters and setters need to be placed.
While placing setters, consider adding validation logic within the setters to ensure these are valid email and phone and follows expected formats.